inc/plugins/thankyoulike.php

```diff
@@ -198,6 +198,11 @@ function thankyoulike_install()
 		$db->query("ALTER TABLE ".TABLE_PREFIX."users ADD `tyl_unumptyls` int(100) NOT NULL default '0'");
 	}
 
+	if(!$db->field_exists('tyl_lastadddeldate', 'users'))
+	{
+		$db->query("ALTER TABLE ".TABLE_PREFIX."users ADD `tyl_lastadddeldate` bigint(30) NOT NULL default '0'");
```

Comment:
Please change the db structure to
```
int unsigned NOT NULL default '0'
```
It's more default mybb like ;)

Reply:
Done!